iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesParserTest.java

```diff
@@ -215,4 +216,75 @@ void shouldNotFailIfEmptyFileAfterEvaluation() throws IOException, URISyntaxExce
       assertThat(logs).contains("Blank evaluated file, skipping processing of Helm file path/to/file.yaml");
     }
   }
+
+  @Test
+  void shouldRemoveLineNumberCommentForNewDocumentAfterEvaluation() throws IOException, URISyntaxException {
```

Comment:
```suggestion
  void shouldNotCrashOnNewDocumentAfterEvaluation() throws IOException, URISyntaxException {
```
IMO, the detailed name this test has now would be more suitable for a unit test of the corresponding post-processor, which you plan to add